opacus/accountants/utils.py

```diff
@@ -13,26 +13,20 @@
 # limitations under the License.
 
 from opacus.accountants import create_accountant
-
-
-# min range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MIN_BOUND = 0.01
-# starting point for a max range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MAX_BOUND = 10
-# condition to halt binary search for sigma given epsilon
-SIGMA_PRECISION = 0.01
-# max possible value for returned sigma.
-# Noise higher than MAX_SIGMA considered unreasonable
-MAX_SIGMA = 2000
+from typing import Optional
 
 
 def get_noise_multiplier(
     *,
     target_epsilon: float,
     target_delta: float,
     sample_rate: float,
-    epochs: int,
+    epochs: Optional[int] = None,
+    steps: Optional[int] = None,
     accountant: str = "rdp",
+    epsilon_tolerance: float = 0.01,
+    sigma_low: float = 0.01,
+    sigma_high: float = 200,
```

Comment:
Why we are so sure that 200 is high enough? 

Why not eliminate this constant by probing `sigma_high = tol*2^k` using this removed algorithm:
```
    sigma_high = tol
    while eps > target_epsilon:
        sigma_high = 2 * sigma_high
        accountant.steps = [(sigma_high, sample_rate, int(epochs / sample_rate))]
        eps = accountant.get_epsilon(delta=target_delta, **kwargs)
        if sigma_high > MAX_SIGMA:
            raise ValueError("The privacy budget is too low.")
```